addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1825,7 +1825,7 @@ export function setTagName(el, newTagName) {
         return el;
     }
     const n = document.createElement(newTagName);
-    if (paragraphRelatedElements.includes(el.nodeName)) {
+    if (el.nodeName !== 'LI') {
```

Comment:
`paragraphRelatedElements` already doesn't include `LI` so how does this prevent the carrying of attributes of `LI` more than it did before? What am I missing?
![image](https://github.com/odoo/odoo/assets/24205914/22a3ab1e-67f2-4a0a-b07e-d63b8b2e1f04)